lib/howl/buffer.moon

```diff
@@ -277,6 +278,19 @@ class Buffer extends PropertyObject
       b_end_pos = @byte_offset end_pos
       @_buffer\ensure_styled_to pos: b_end_pos
 
+  mode_at: (pos) =>
```

Comment:
Looking now it seems `buffer.mode` is clear, but `buffer.config` still shows occurrences that just uses the plain `buffer.config`. E.g. from doing an `ag 'buffer\.config'`:

- text.moon
- auto_pair.moon
- formatting.moon
- [..]